Model/Builders/ApiBuilder.php

```diff
@@ -94,6 +95,58 @@ public function get(int $storeId = null, string $testApiKey = null)
         return $gingerClient;
     }
 
+    /**
+     * Get function
+     *
+     * @param int $storeId
+     * @param string $testApiKey
+     *
+     * @return bool|\Ginger\ApiClient
+     * @throws \Exception
+     */
+    public function getApiClient(int $storeId = null, string $testApiKey = null)
+    {
```

Comment:
Could you explain the difference between this method and the get() method from this class?
Looks like both of them doing the same thing but in slightly different way